frontend/src/components/ViewTrips/trip.test.js

```diff
@@ -4,21 +4,22 @@ import 'firebase/firebase-firestore';
 import { getDateRange } from './trip.js'
 
 test('getDateRange test', () => {
+  // Dates used for both test and expected date strings
```

Comment:
Nit: add period, and clarify the input month is off-by-one given the zero indexing of JS dates.

Reply:
Fixed.